packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -102,6 +102,19 @@ export interface RemoteVideoTileMenuOptions {
    */
   isHidden?: boolean;
 }
+
+/* @conditional-compile-remove(click-to-call) */
+/**
+ * Options for the local video tile in the Call composite.
+ *
+ * @beta
+ */
+export interface LocalVideoTileOptions {
+  /**
+   * Position of the local video tile. If unset will render the local tile in the floating local position.
+   */
+  position?: 'grid' | 'floating' | 'hidden';
```

Comment:
what does grid refer to with respect to grid vs overflow gallery?

Reply:
And does floating mean it will always be floating and never docked anymore?

Reply:
So this might be a naming issue with the videoGallery layouts but it will follow the `default layout` when `grid` is selected. so anything that that local tile would do in the default layout will be true here.